### ISSUE TITLE
Single Slider 

### ISSUE THREAD

**Issue by usmantarar51** [WorldDownTown/RangeSeekSlider#50]
Can we add single left slider only ?

**Comment by ChitvanSaxena** [comment]
yes you can do that by simply turning "On" the "Disable Range" option in Attribute inspector. Remember the visible Handle acts as a representation for Max Value.
